pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -141,6 +141,7 @@ type EndpointPolicyDirection struct {
 	Removing AllowedIdentityList `json:"removing,omitempty"`
 	// Deprecated
 	Adding AllowedIdentityList `json:"adding,omitempty"`
+	State  string              `json:"state,omitempty"`
```

Comment:
Bikeshedding: `state` generally implies a machine-readable enumeration, rather than a freeform string. Perhaps this could be something like
```go
type EndpointPolicyState string

const (
    EndpointPolicyStateEnforcing EndpointPolicyState = "enforcing"
    EndpointPolicyStateNonEnforcing EndpointPolicyState = "non-enforcing"
    EndpointPolicyStateDisabled EndpointPolicyState = "disabled"
)
```

Reply:
Yes, agreed. An enum would probably be better suitable for this case.

Reply:
Ok. Will do